### ISSUE TITLE
Default template package.json doesn't support NPM 3

### ISSUE THREAD

**Issue by chipbell4** [SpringRoll/SpringRollStudio#12]
When installing from the default template, it only installs `grunt` and `grunt-springroll`. I'm thinking there's some peer dependencies that are now being ignored due to changes between NPM 2 and 3, and so after an `npm install` with npm 3 I get errors at build time:

```
>> Local Npm module "grunt-bower-task" not found. Is it installed?
>> Local Npm module "grunt-concat-json" not found. Is it installed?
>> Local Npm module "grunt-contrib-clean" not found. Is it installed?
>> Local Npm module "grunt-contrib-concat" not found. Is it installed?
>> Local Npm module "grunt-contrib-connect" not found. Is it installed?
>> Local Npm module "grunt-contrib-copy" not found. Is it installed?
>> Local Npm module "grunt-contrib-jshint" not found. Is it installed?
>> Local Npm module "grunt-contrib-less" not found. Is it installed?
>> Local Npm module "grunt-contrib-uglify" not found. Is it installed?
>> Local Npm module "grunt-contrib-watch" not found. Is it installed?
>> Local Npm module "grunt-newer" not found. Is it installed?
>> Local Npm module "grunt-simple-version" not found. Is it installed?
>> Local Npm module "grunt-springroll-helper" not found. Is it installed?
>> Local Npm module "grunt-text-replace" not found. Is it installed?
Warning: Task "clean:main" not found. Use --force to continue.
```

I'm guessing we need to be explicit about those dependencies?

**Comment by probityrules** [comment]
I think this issue is the same issue described here for `grunt-springroll`: https://github.com/SpringRoll/grunt-springroll/issues/5

**Comment by chipbell4** [comment]
Resolved in [grunt-springroll 0.7.11](https://www.npmjs.com/package/grunt-springroll)
